packages/core/src/App/Containers/OneTimeDepositModal/one-time-deposit-modal.tsx

```diff
@@ -0,0 +1,114 @@
+    } = ui;
+    const liveChat = useLiveChat(false, loginid);
+    const has_mf_account_deposited = useHasMFAccountDeposited();
+
+    React.useEffect(() => {
+        if (has_mf_account_deposited) {
+            onCloseModal();
+        }
+    }, [has_mf_account_deposited]);
+
+    const onLiveChatClick = () => {
+        liveChat.widget?.call('maximize');
+    };
+
+    const onCloseModal = () => {
+        setShouldShowOneTimeDepositModal(false);
+        toggleAccountSuccessModal();
+    };
+
+    const getModalContent = () => (
+        <div className='one-time-deposit-modal__content'>
+            <div className='one-time-deposit-modal__title'>
+                <Text as='h1' size={is_mobile ? 'm' : 'l'} weight='bold'>
+                    <Localize i18n_default_text='Deposit' />
+                </Text>
+                <Text as='p' size={is_mobile ? 'xs' : 's'} align='center'>
+                    <Localize i18n_default_text='Account created. Select payment method for deposit.' />
+                </Text>
+            </div>
+            <div className='one-time-deposit-modal__description'>
+                <InlineMessage
+                    type='information'
+                    size='sm'
+                    message={localize(
+                        'We don’t charge deposit fees! Once your account is verified, you will be able to trade, make additional deposits, or withdraw funds.'
+                    )}
+                />
+                <Button
+                    className='one-time-deposit-modal__description--livechat'
+                    data-testid='dt_live_chat'
+                    onClick={onLiveChatClick}
+                    transparent
+                    type='button'
+                >
+                    <Icon
+                        color='red'
+                        icon='IcLiveChat'
+                        className='one-time-deposit-modal__description--livechat-icon'
+                    />
+                    <Text color='loss-danger' size='xs' weight='bold'>
+                        {localize('Live chat')}
+                    </Text>
+                </Button>
+            </div>
+            <DepositFiatIframe />
+        </div>
+    );
+
+    return (
+        <React.Fragment>
+            <DesktopWrapper>
+                <Modal
+                    className='one-time-deposit-modal'
+                    is_open={should_show_one_time_deposit_modal}
+                    title=' '
+                    toggleModal={onCloseModal}
+                    height='auto'
+                    has_close_icon
+                >
+                    <Modal.Body className='one-time-deposit-modal__body'>{getModalContent()}</Modal.Body>
+                </Modal>
+            </DesktopWrapper>
+            <MobileWrapper>
+                <MobileFullPageModal
+                    className='one-time-deposit-modal'
+                    body_className='one-time-deposit-modal__body'
+                    header=' '
```

Comment:
do we need to pass header? It is optional in types?

Reply:
Yes we need to pass in this header if not we can't show the modal header with close button.